framework/tests/value-no-overflow/ValueNoOverflow.java

```diff
@@ -0,0 +1,31 @@
+import org.checkerframework.common.value.qual.*;
+
+class ValueNoOverflow {
```

Comment:
Please add more tests.  You could copy checker/tests/index/Transfer*.java and replace the LBC annotations with the proper `@IntRange` annotation.  You should also test with longs rather than ints.  Also, please a few tests with bytes and shorts.